<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ReportAgeClassificationSetupDialog</class>
 <widget class="QDialog" name="ReportAgeClassificationSetupDialog">
  <property name="windowModality">
   <enum>Qt::ApplicationModal</enum>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>352</width>
    <height>217</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="windowTitle">
   <string>параметры отчёта</string>
  </property>
  <property name="sizeGripEnabled">
   <bool>true</bool>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="7" column="1">
    <spacer name="verticalSpacer">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>40</height>
      </size>
     </property>
    </spacer>
   </item>
   <item row="2" column="0">
    <widget class="QLabel" name="lblEndDate">
     <property name="text">
      <string>Дата окончания периода</string>
     </property>
    </widget>
   </item>
   <item row="4" column="1">
    <widget class="QComboBox" name="cmbPrimary">
     <item>
      <property name="text">
       <string/>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Первичный</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Вторичный</string>
      </property>
     </item>
    </widget>
   </item>
   <item row="3" column="1">
    <widget class="CPersonComboBoxEx" name="cmbPerson"/>
   </item>
   <item row="1" column="0">
    <widget class="QLabel" name="lblBegDate">
     <property name="text">
      <string>Дата начала периода</string>
     </property>
    </widget>
   </item>
   <item row="4" column="0">
    <widget class="QLabel" name="lblPrimary">
     <property name="text">
      <string>Первичность</string>
     </property>
    </widget>
   </item>
   <item row="1" column="1">
    <widget class="CDateEdit" name="edtBegDate"/>
   </item>
   <item row="9" column="0" colspan="2">
    <widget class="QDialogButtonBox" name="buttonBox">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="standardButtons">
      <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
     </property>
    </widget>
   </item>
   <item row="2" column="1">
    <widget class="CDateEdit" name="edtEndDate"/>
   </item>
   <item row="3" column="0">
    <widget class="QLabel" name="lblPerson">
     <property name="text">
      <string>&amp;Врач</string>
     </property>
     <property name="buddy">
      <cstring>cmbPerson</cstring>
     </property>
    </widget>
   </item>
   <item row="0" column="0" colspan="2">
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QLabel" name="lblMKB">
       <property name="text">
        <string>Коды диагнозов по &amp;МКБ</string>
       </property>
       <property name="buddy">
        <cstring>cmbMKBFilter</cstring>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QComboBox" name="cmbMKBFilter">
       <item>
        <property name="text">
         <string>Игнор.</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Интервал</string>
        </property>
       </item>
      </widget>
     </item>
     <item>
      <widget class="CICDCodeEdit" name="edtMKBFrom">
       <property name="enabled">
        <bool>false</bool>
       </property>
       <property name="sizePolicy">
        <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="maximumSize">
        <size>
         <width>40</width>
         <height>16777215</height>
        </size>
       </property>
       <property name="inputMask">
        <string>a00.00; </string>
       </property>
       <property name="text">
        <string>A.</string>
       </property>
       <property name="maxLength">
        <number>6</number>
       </property>
      </widget>
     </item>
     <item>
      <widget class="CICDCodeEdit" name="edtMKBTo">
       <property name="enabled">
        <bool>false</bool>
       </property>
       <property name="sizePolicy">
        <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="maximumSize">
        <size>
         <width>40</width>
         <height>16777215</height>
        </size>
       </property>
       <property name="inputMask">
        <string>a00.00; </string>
       </property>
       <property name="text">
        <string>Z99.9</string>
       </property>
       <property name="maxLength">
        <number>6</number>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item row="5" column="0">
    <widget class="QLabel" name="lblEventType">
     <property name="text">
      <string>&amp;Тип обращения</string>
     </property>
     <property name="buddy">
      <cstring>cmbEventType</cstring>
     </property>
    </widget>
   </item>
   <item row="5" column="1">
    <widget class="CRBComboBox" name="cmbEventType"/>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>CPersonComboBoxEx</class>
   <extends>QComboBox</extends>
   <header>Orgs/PersonComboBoxEx.h</header>
  </customwidget>
  <customwidget>
   <class>CICDCodeEdit</class>
   <extends>QLineEdit</extends>
   <header>library/ICDCodeEdit.h</header>
  </customwidget>
  <customwidget>
   <class>CRBComboBox</class>
   <extends>QComboBox</extends>
   <header>library/crbcombobox.h</header>
  </customwidget>
  <customwidget>
   <class>CDateEdit</class>
   <extends>QDateEdit</extends>
   <header>library/DateEdit.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>ReportAgeClassificationSetupDialog</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>132</x>
     <y>650</y>
    </hint>
    <hint type="destinationlabel">
     <x>65</x>
     <y>418</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>ReportAgeClassificationSetupDialog</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>245</x>
     <y>650</y>
    </hint>
    <hint type="destinationlabel">
     <x>267</x>
     <y>420</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
